src/main/features/response/views/statement-of-means/maintenance.njk

```diff
@@ -10,30 +10,30 @@
       <form method="post" novalidate
         class="analytics-submit-event-trigger"
         data-event-action="SoM: Do you or your partner pay maintenance for any children?"
-        data-event-label-from="option">
+        data-event-label-from="declared">
```

Comment:
another GA label bit

Reply:
The same, declared is just value source.